workshops/gloo-mesh-demo/install/istio/gm-ingress-gateway.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: admin.gloo.solo.io/v2
+kind: GatewayLifecycleManager
+metadata:
+  name: istio-ingressgateway
+  namespace: gloo-mesh
+spec:
+  installations:
+      # The revision for this installation, such as 1-14
+    - gatewayRevision: 1-15
+      # List all workload clusters to install Istio into
+      clusters:
+      - name: cluster1
+        # If set to true, the spec for this revision is applied in the cluster
+        activeGateway: true
+      - name: cluster2
+        activeGateway: true
+      istioOperatorSpec:
+        # No control plane components are installed
+        profile: empty
+        # Solo.io Istio distribution repository; required for Gloo Istio.
+        # You get the repo key from your Solo Account Representative.
+        hub: us-docker.pkg.dev/gloo-mesh/istio-workshops
+        # The Solo.io Gloo Istio tag
+        tag: 1.15.1-solo
+        components:
+          ingressGateways:
+          # Enable the default ingress gateway
+            - name: istio-ingressgateway
+              # Deployed to gloo-mesh-gateways by default
+              namespace: istio-gateways
+              enabled: true
+              label:
```

Comment:
we might need to be careful about setting the labels and selectors here, this could cause problems with canary deployments as the deployment/services might point to both

Reply:
Are you suggesting that we remove the label section entirely?